packages/wonder-blocks-dropdown/components/multi-select.js

```diff
@@ -174,41 +157,69 @@ export default class MultiSelect extends React.Component<Props, State> {
                 // If there is one item selected, we display its label. If for
                 // some reason we can't find the selected item, we use the
                 // display text for the case where nothing is selected.
-                const selectedItem = items.find(
-                    (item) => item.value === selectedValues[0],
+                const selectedItem = React.Children.toArray(children).find(
+                    (option) => option.props.value === selectedValues[0],
                 );
-                return selectedItem ? selectedItem.label : noSelectionText;
-            case items.length:
+                return selectedItem
+                    ? selectedItem.props.label
+                    : noSelectionText;
+            case React.Children.count(children):
                 return `All ${selectItemType}`;
             default:
                 return `${selectedValues.length} ${selectItemType}`;
         }
     }
 
-    getMenuItems() {
-        const {items, selectedValues} = this.props;
-        const menuItems = items.map((item, index) => {
-            return (
-                <OptionItem
-                    disabled={item.disabled}
-                    key={item.value}
-                    label={item.label}
-                    /* eslint-disable-next-line react/jsx-handler-names */
-                    onClick={item.onClick}
-                    onToggle={(value, state) =>
-                        this.handleSelected(value, state)
-                    }
-                    selected={selectedValues.includes(item.value)}
-                    value={item.value}
-                    variant="checkbox"
-                />
-            );
+    getShortcuts(): Array<
+        React.Element<typeof ActionItem | typeof SeparatorItem>,
+    > {
+        const {children, selectedValues, shortcuts} = this.props;
+        const numOptions = React.Children.count(children);
+        // TODO(sophie): translate for i18n
```

Comment:
This almost feels like a STOPSHIP... we need to figure this out soon otherwise people will start using them and then we'll get complaints about English on i18n sites.

Reply:
I think there may be another English string somewhere.  It would be nice to have a linter to find them.

Reply:
I agree... We can potentially use props to pass in all the phrasing that we want, but we would need to be wary of phrasal ordering assumptions.